src/containers/find-offer/filter-bar-menu/FilterBarMenu.constants.ts

```diff
@@ -0,0 +1,38 @@
+import { CardsViewEnums } from '~/types/findOffers/enums/viewSwitcher.enums'
+
+
+export const switcherOptions = {
+  left: {
+    text: 'findOffers.topMenu.tutorsOffers',
+    tooltip: 'findOffers.contentSwitcher.switcher-tutor'
+  },
+  right: {
+    text: 'findOffers.topMenu.studentsRequests',
+    tooltip: 'findOffers.contentSwitcher.switcher-student'
+  }
+}
+  
+export const sortByFields = [
+  {
+    value: 'popularity',
+    title: 'filters.sortBy.sortByFields.popularity'
+  },
+  {
+    value: 'newest',

```

Comment:
those values needed for sorting offers on backEnd check how it should be named

Reply:
does we have sorting service of offers on Back End?

Reply:
values should be named as fields in model